hw-model/c-binding/examples/api/caliptra_fuses.h

```diff
@@ -0,0 +1,19 @@
+// Licensed under the Apache-2.0 license
+#ifndef CALIPTRA_FUSES_H
+#define CALIPTRA_FUSES_H
+
+#include "caliptra_api.h"
+
+#define CALIPTRA_FUSE_WRITE(model, offset, data) \
```

Comment:
Do these need to be macros, or could we get away with inline functions?

Reply:
done